### ISSUE TITLE
cannot get CCN in such situation

### ISSUE THREAD

**Issue by qwertmy** [terryyin/lizard#194]
xxx.c
void func_a()
{
       xxx;
       xxx;
       #if 0
           if (a==b       /****error compile****/
           {
                 cccccc;
           }
      #endif
      xxx;
}
================================================
  NLOC    CCN   token  PARAM  length  location  
------------------------------------------------
1 file analyzed.
==============================================================
NLOC    Avg.NLOC  AvgCCN  Avg.token  function_cnt    file
--------------------------------------------------------------
     23       0.0     0.0        0.0         0     1.c

when the error compile recovered
================================================
  NLOC    CCN   token  PARAM  length  location  
------------------------------------------------
      23      9    114      1      30 XXX@8-37@1.c
1 file analyzed.
==============================================================
NLOC    Avg.NLOC  AvgCCN  Avg.token  function_cnt    file
--------------------------------------------------------------
     23      23.0     9.0      114.0         1     1.c

**Comment by terryyin** [comment]
Yes, lizard cannot accurately process preprocessor. That's a trade off so that it doesn't need all the header files.

I'm considering implementing to support at least preprocessing within the file...

**Comment by terryyin** [comment]
After some consideration, I decided that putting code block in "#if 0" is a bad practice. We should rely on our version control system for such a need. So I will still delay the implementation of real preprocessing support.
